setup.py

```diff
@@ -46,8 +46,10 @@
       scripts=glob('scripts/*'),
       extras_require={'test': ["nose >= 0.10.1", "pep8"],
                       'doc': ["Sphinx >= 1.2.2", "sphinx-bootstrap-theme"]},
-      install_requires=['click', 'numpy >= 1.7',
+      install_requires=['click == 7.0-dev', 'numpy >= 1.7',
```

Comment:
Okay. I suspect the API will be stable enough given the history of the project, but we should get this revised as soon as we can. I'm not sure if setup.py can have a dependency on a specific commit, which would be nice if easily/cleanly supported. My guess is that 7.0 will be out relatively soon given the prior releases of click